### PR TITLE
added Slider editor

### PR DESCRIPTION
**Pull request by gkostov**
I needed a slider for a field and made this extension. Thought to share if anyone might need the same.

### CODE REVIEW

**Comment by lorvent** [comment]
Hello,
Do you have any working demo?

File: src/inputs-ext/slider/slider.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            
+//            // copy `input-sm | input-lg` classes to placeholder input
+//            if($.fn.editableform.engine === 'bs3') {
+//                if(this.$input.hasClass('input-sm')) {
+//                    this.$input.siblings('input.tt-hint').addClass('input-sm');
+//                }
+//                if(this.$input.hasClass('input-lg')) {
+//                    this.$input.siblings('input.tt-hint').addClass('input-lg');
+//                }
+//            }
```
Comment (burzum) [inline src/inputs-ext/slider/slider.js]: Can you remove the unnecessary commented code?

Comment (burzum) [inline src/inputs-ext/slider/slider.js]: Can you remove the unnecessary commented code?

Comment (gkostov) [inline src/inputs-ext/slider/slider.js]: Sure, missed that one :-)
Cheers

**Comment by lorvent** [comment]
can you provide a jsfiddle demo for it?

**Comment by gkostov** [comment]
Here is it https://jsfiddle.net/7g2912q5/2/
It reminded me that the slider plugin is that from http://www.eyecon.ro/bootstrap-slider/ . I've used a fork in the fiddle because the original files could not be loaded there.

**Comment by lorvent** [comment]
Thanks for that :+1:

**Pull request merged**